Tools/TestWebKitAPI/Tests/WebKitCocoa/PDFSnapshot.mm

```diff
@@ -173,7 +173,7 @@
         EXPECT_NE(page.get(), nil);
 
         EXPECT_TRUE(CGRectEqualToRect([page bounds], CGRectMake(0, 0, 800, 14400)));
-        EXPECT_TRUE([[page colorAtPoint:NSMakePoint(400, 300)] isEqual:[CocoaColor whiteColor]]);
+        EXPECT_TRUE([[page colorAtPoint:NSMakePoint(400, 300)] isEqual:[CocoaColor colorWithRed:1.0 green:1.0 blue:1.0 alpha:1.0]]);
```

Comment:
`compareColors` helper would make this less error prone and let you still use `whiteColor`

Reply:
And if you added Gtest assertion support, you could write:

```
EXPECT_EQ([page colorAtPoint:NSMakePoint(400, 300)], [CocoaColor whiteColor]);
```

... which would end up calling `compareColors` for you under the hood!